docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.8"
+services:
+  backend:
```

Comment:
You should pass correct environment variables for the backend service. It should be at least url (which will be different from your application.yml specified. Read about using another docker compose service as part of the url